ci/ci/build.py

```diff
@@ -359,7 +359,11 @@ def cleanup(self, batch, scope, parents):
 gcloud -q auth activate-service-account \
   --key-file=/secrets/gcr-push-service-account-key/gcr-push-service-account-key.json
 
-gcloud -q container images untag {shq(self.image)}
+until {{ gcloud -q container images untag {shq(self.image)} || ! gcloud -q container images describe {shq(self.image)} }}
```

Comment:
Why are the outermost braces necessary here?  And I feel like you need a semicolon, although maybe I don't understand this meaning of curly brace. My bash skills were never very good.

Reply:
The curlies aren't necessary, I am also bash-naive and was trying to be extra careful.

I don't want a semicolon. I want to run through this loop until either the untag succeeds or the describe does not succeed (i.e. the image does not exist).